app/models/veteran.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+require 'evss/auth_headers'
+
+# Veteran model
+class Veteran < Common::RedisStore
```

Comment:
I would have expected an error if you don't define the namespace and key in your model (looking at the redis_store.rb model)

Reply:
hey @dhall  I see some models like user.rb that set those, and some that dont like dependents_application.rb. I am also wondering if we really need to store in redis yet. What do you think?

Reply:
ah, looks like including RedisForm sets some values for you. I don't think that's what you want though since this isn't form related (that I'm aware of). I'm open to what you think makes sense but generally I take the approach of not prematurely optimizing so I'd likely hold off on redis until we know more about usage and performance.